### ISSUE TITLE
Add a command for moving a tab right/left

### ISSUE THREAD

**Issue by mattiasJohnson** [jupyterlab/jupyterlab#13800]
### Problem

There is no command for moving a tab right/left.

### Proposed Solution
Adding a command for moving a tab right/left.
 

### Additional context
There are many jupterlab users who prefer to use keyboard shortcuts and it would be great being able to move the tabs around without having to switch to the mouse. This is present in many other development environments such as for example vscode.

**Comment by JasonWeill** [comment]
For example, in Firefox (on a US keyboard), one can move the current tab left or right with <kbd>CTRL</kbd>+<kbd>SHIFT</kbd>+<kbd>PGUP</kbd> or <kbd>PGDN</kbd> respectively.

**Comment by mattiasJohnson** [comment]
I am looking for a command so I can create a keyboard binding for moving a Jupyterlab tab left/right. Maybe I wasn't clear enough because if I am not mistaken this moves the Firefox tab?

**Comment by krassowski** [comment]
All is good, in triage meeting we were just thinking what would be a good default keybindging for that command in JupyterLab (if any), hence the note on what is the default in FF

**Comment by mattiasJohnson** [comment]
Yes sorry about that! I should have realised you just gave an example I read it too hastily
